src/qt/glcanvas.cpp

```diff
@@ -403,8 +435,15 @@ bool wxGLCanvas::Create(wxWindow *parent,
                         const wxString& name,
                         const wxPalette& palette)
 {
-    wxLogError("Missing implementation of " + wxString(__FUNCTION__));
-    return false;
+    const int* attrsList = dispAttrs.GetGLAttrs();
+
+    if (!attrsList)
+    {
+        wxFAIL_MSG("wxGLAttributes object is empty.");
+        return false;
+    }
```

Comment:
Minor, but could be just

```suggestion
    wxCHECK_MSG(attrsList, false, "wxGLAttributes object is empty.");
```